packages/stores/src/account/index.ts

```diff
@@ -815,6 +815,91 @@ export class OsmosisAccountImpl {
     );
   }
 
+  async sendAddToConcentratedLiquidityPositionMsg(
+    positionId: string,
+    amount0: string,
+    amount1: string,
+    maxSlippage = "2.5",
+    memo: string = "",
+    onFulfill?: (tx: any) => void
+  ) {
+    // refresh position
+    const queryPosition =
+      this.queries.queryLiquidityPositionsById.getForPositionId(positionId);
+    await queryPosition.waitFreshResponse();
+
+    const amount0WithSlippage = new Dec(amount0)
+      .mul(new Dec(1).sub(new Dec(maxSlippage).quo(new Dec(100))))
+      .truncate()
+      .toString();
+    const amount1WithSlippage = new Dec(amount1)
+      .mul(new Dec(1).sub(new Dec(maxSlippage).quo(new Dec(100))))
+      .truncate()
+      .toString();
+
+    const aminoMsg = {
+      type: this._msgOpts.clAddToConcentratedPosition.type,
+      value: {
+        position_id: positionId,
+        sender: this.base.bech32Address,
+        amount0: amount0,
+        amount1: amount1,
+        token_min_amount0: amount0WithSlippage,
+        token_min_amount1: amount1WithSlippage,
+      },
+    };
+
+    const protoMsg = {
+      typeUrl: "/osmosis.concentratedliquidity.v1beta1.MsgAddToPosition",
```

Comment:
do we have a constants file for endpoints like this?

Reply:
Closest thing may be msg-opts.ts? But I think those are just used for Amino URLs and gas settings. Open to answering Qs for what those are.